projects/1.20/assets/table-top-craft/table_top_craft/lang/zh_cn-base.json

```
@@ -0,0 +1,164 @@
+{
+"_comment1": "Item Groups",
+	"itemGroup.table_top_craft.tab": "Table Top Craft",
+	
+"_comment2": "Blocks",
+	"block.table_top_craft.chess_piece_figure": "棋像",
+	"block.table_top_craft.tic_tac_toe": "井字棋",
+
+	"_comment3": "GUI",
+	"gui.table_top_craft.chess.board_settings": "棋盘设置",
+	"gui.table_top_craft.chess.board_colors": "棋盘颜色",
+	"gui.table_top_craft.chess.board_pieces": "棋盘棋子",
+	"gui.table_top_craft.chess.board_evaluator": "棋盘评估器",
+	"gui.table_top_craft.piece_figure.piece_settings": "棋像设置",
+	"gui.table_top_craft.piece_figure.toggle_rotation": "切换棋子旋转与否",
+	"gui.table_top_craft.chess.fen_loader": "FEN加载器",
+	"gui.table_top_craft.chess.move_log": "棋局记录",
+	"gui.table_top_craft.chess.move_log.white": "白",
+	"gui.table_top_craft.chess.move_log.black": "黑",
+	"gui.table_top_craft.chess.button.settings": "设置",
+	"gui.table_top_craft.chess.button.colors": "颜色",
+	"gui.table_top_craft.chess.button.pieces": "棋子",
+	"gui.table_top_craft.chess.new_game": "新游戏",
+	"gui.table_top_craft.chess.copy_fen": "复制FEN",
+	"gui.table_top_craft.chess.load_fen": "加载FEN",
+	"gui.table_top_craft.chess.evaluate": "棋局评估",
+	"gui.table_top_craft.chess.cancel": "取消",
+	"gui.table_top_craft.chess.back": "返回",
+	"gui.table_top_craft.chess.confirm_fen": "确认FEN",
+	"gui.table_top_craft.chess.confirm_color": "确认颜色",
+	"gui.table_top_craft.chess.confirm_colors": "确认颜色",
+	"gui.table_top_craft.chess.show_tile_info": "显示棋格信息",
+	"gui.table_top_craft.chess.show_available_moves": "显示可移动格",
+	"gui.table_top_craft.chess.show_previous_move": "显示上一步棋",
+	"gui.table_top_craft.chess.evaluation.white.mobility": "白方移动能力：%d",
+	"gui.table_top_craft.chess.evaluation.white.king_threats": "白方国王威胁：%d",
+	"gui.table_top_craft.chess.evaluation.white.possible_attacks": "白方进攻可能性：%d",

```

[考量]看key这里似乎计算的是种类数？如果如此，那么“可能性”直接计数或许有点怪

```suggestion
	"gui.table_top_craft.chess.evaluation.white.possible_attacks": "白方进攻可能：%d",
```

```suggestion
	"gui.table_top_craft.chess.evaluation.white.possible_attacks": "白方进攻可能数：%d",
```

---

经检验，是当前棋局中某一方可以一步吃掉的对手棋子的数量，改成下面那个了